humilis_batch/resources.yaml.j2

```diff
@@ -109,6 +110,17 @@ resources:
                 {% endfor %}
             {% endif %}
 
+    {% if has_launch_template %}
```

Comment:
you could replace this by `{% if user_data %}` making `has_launch_template` unnecessary.